code/modules/modular_computers/computers/item/computer_ui.dm

```diff
@@ -137,14 +137,14 @@
 	switch(action)
 		if("PC_exit")
 			//you can't close apps in emergency mode.
-			if(internal_cell.charge)
+			if(isnull(internal_cell) || internal_cell.charge)
```

Comment:
```suggestion
			if(internal_cell?.charge)
```
You had the `?` initially, why not just use that?

Reply:
if(internal_cell?.charge) evaluates false if the internal_cell is null. It should be true if the internal_cell is null, or internal_cell is not null but has a charge of 0.